enclave_wrangler/objects_api.py

```diff
@@ -354,6 +355,10 @@ def fetch_cset_and_member_objects(
                              f'for that specific container.')
         cset_containers.append(container[0]['properties'])
 
+    # Returning None if no data
+    if not cset_containers and not cset_versions:
+        return None
```

Comment:
I actually realized that `return None` is the same as `return`. They both return `None`. So I shortened this to just `return`.